views/elements/message_box.tmpl

```diff
@@ -2,8 +2,13 @@
 {{with .}}
 <h2>Send a Message</h2>
 <hr>
-<form id="studiomessage">
-  <textarea class="form-control" id="comments" name="comments" rows="4" placeholder="Why not send a message to the studio presenters? Have a bit of banter, share your opinions or simply request your favourite song?"></textarea>
+<form id="studiomessage" action="/" method="post">
+  <textarea class="form-control" id="comments" name="message" rows="4" placeholder=
+  {{if .MsgBoxPrompt}}
+  	"Message cannot be sent at this time. Is Jukebox on Air?"
```

Comment:
"Is Jukebox on Air?" is a bit of an odd message, the user certainly knows, and the website should probably as well

Perhaps the box could be hidden if this were the case?

Reply:
Not sure about hiding the box, as this would both change the page layout and become a problem if a show then starts
Just copied the messages from Brooke's implementation, not sure what else to put in there

Reply:
Yeah I just copied the radio player implementation. We could disable the input, and say something like no shows are on air

Reply:
I believe my new radio player branch has a better way of doing this in JS if that's still needed :)